docs/pages/usekliveregion.vue

```diff
@@ -84,9 +84,12 @@
       <p>
         Sending an assertive message updates the text content of
         <code>aria-live="assertive"</code> region.
-        <em>It should be used with caution because it disrupts the user's flow. Use it only to send
-          messages that require immediate attention, such as errors.</em>
+        <em>Use this only for messages that require immediate attention, such as critical errors or
```

Comment:
Yes, thank you - I think this is a nice re-formulation that also resolves the issue.